### ISSUE TITLE
↔️ Create a mockup for the settings-block under "Appearance and accessibility"

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#40245]
mockup/design for #40241

**Comment by szaimen** [comment]
Why was this ticket created? To better track the different sub-tasks?

**Comment by szaimen** [comment]
Hi, here is the MVP mockup for this feature:

![image](https://github.com/nextcloud/server/assets/42591237/c9a4ead1-b3f9-4cc6-945a-16e0ead7c3db)

It should be put into the `Appearance and Accessibility` settings section at the very bottom (below the keyboard shortcuts section).

Let me know what you think! :)

---

For reference here also a mockup how the feature could look like in the very end if there should be enough time:

<details>
<summary>click here to expand</summary>

![image](https://github.com/nextcloud/server/assets/42591237/f8ce776b-f3c3-49fe-8a4f-b66e9ec196a2)

This would then have an additional admin section in theming where we would show this switch.

</details>

**Comment by sorbaugh** [comment]
Hi @skjnldsv ,

question came up yesterday if we need new Vue components to implement this or do we already have them? 🤔

**Comment by skjnldsv** [comment]
There might be a few we can already use, but that doesn't matter tbh, the whole UI needs to be built from scratch

**Comment by susnux** [comment]
We have a similar use case at the forms app where we implemented something similar for question order, so probably we can reuse that :)

**Comment by come-nc** [comment]
> We have a similar use case at the forms app where we implemented something similar for question order, so probably we can reuse that :)

There is the same thing in table view creation UI as well: https://github.com/nextcloud/tables/blob/main/src/modules/main/partials/editViewPartials/SelectedViewColumns.vue
